src/VisualStudio/Core/Def/PackageRegistration.pkgdef

```diff
@@ -13,3 +13,8 @@
 "CS"="CA719A03-D55C-48F9-85DE-D934346E7F70"
 "VB"="EEC3DF0D-6D3F-4544-ABF9-8E26E6A90275"
 
+[$RootKey$\FeatureFlags\Roslyn\LSP\Editor]
+"Description"="Enables the LSP powered C#/VB editing experience outside of CodeSpaces."
+"Value"=dword:00000000
+"Title"="Enable experimental C#/VB editor (requires restart)"
+"PreviewPaneChannels"="Preview,int.main"
```

Comment:
I assume we don't want to expose this to external customers correct? Just double checking because there's a separate process for that 😄

Reply:
@NTaylorMullen yeah - I think this should just be IntPreview instead of Preview - but is there something more I need to do to restrict it to internal?

Reply:
> but is there something more I need to do to restrict it to internal?

So by default everything can be controlled programmatically for internal or external; however, the "internal" only aspect of feature flags only applies to the Preview Feature panel UI. Meaning, you wont be able to ask users to go and switch the flag on unless you go through a separate process to allow it there for external folks.

>  I do need this to run on integration test machines,

You can still programmatically toggle these feature flags on any box which is what I imagine you're doing for the integration test machines?

-------

Enabling the feature flag checkbox in the preview features panel for external folks docs: https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/3016/How-to-get-on-the-Preview-Features-Page?anchor=enabling-feature-flags-in-release-channel

Reply:
>You can still programmatically toggle these feature flags on any box which is what I imagine you're doing for the integration test machines?
Yeah I am setting the registry key for the feature flag on the integration test machines.  So I'll switch this to int preview